### PR TITLE
Patching package certifi in Tensforflow 2.8 GPU and CPU images

### PR DESCRIPTION
**Pull request by kenny-ezirim**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Description
Patch package certifi in GPU and CPU images of TF 2.8 
### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tensorflow/inference/docker/2.8/py3/Dockerfile.cpu
Comment (shantanutrip) [inline tensorflow/inference/docker/2.8/py3/Dockerfile.cpu:94]: On line [106](https://github.com/aws/deep-learning-containers/pull/2538/files#diff-d17c4581f9b288b81580b22413774fc2b8c706090ac4931f9c94a23550d41380L105), can we please change the version to`tensorflow-serving-api=="2.8.4"`? 

Thanks!!

File: tensorflow/inference/docker/2.8/py3/cu112/Dockerfile.gpu
Comment (shantanutrip) [inline tensorflow/inference/docker/2.8/py3/cu112/Dockerfile.gpu:140]: Can we change [this](https://github.com/aws/deep-learning-containers/pull/2538/files#diff-256c9f375cceea45f3bf8d650a757d33b24004fafe83a0dfc9bcf52a37acee30R152) to tensorflow-serving-api-gpu=="2.8.4"?

File: tensorflow/buildspec-2-8.yml
Comment (shantanutrip) [inline tensorflow/buildspec-2-8.yml:9]: We do not need to comment this section. Commenting the training portions under [images](https://github.com/aws/deep-learning-containers/pull/2538/files#diff-3af2656db81add8a43df467b23e486c7373b1034eed776f4fbb409c1353b5ce2L43) is enough. Please revert this change before merging.

**Review by shantanutrip — COMMENTED** [review]
LGTM!! Please revert the TOML and the buildpsec changes.

File: tensorflow/buildspec-2-8.yml
Comment (kenny-ezirim) [reply tensorflow/buildspec-2-8.yml:9]: Arjuna would say differently - to comment it out.

Comment (shantanutrip) [reply tensorflow/buildspec-2-8.yml:9]: Commenting these sections when not required might lead to delays since we would need to get reviews from the respective code owners.

**Review by shantanutrip — APPROVED** [review]

**Pull request merged**
